config/default/Configuration.h

```diff
@@ -1241,9 +1392,38 @@
  */
 //#define SENSORLESS_PROBING
 
-//
-// For Z_PROBE_ALLEN_KEY see the Delta example configurations.
-//
+/**
+ * Allen key retractable z-probe as seen on many Kossel delta printers - https://reprap.org/wiki/Kossel#Automatic_bed_leveling_probe
+ * Deploys by touching z-axis belt. Retracts by pushing the probe down.
+ */
+//#define Z_PROBE_ALLEN_KEY
+
+#if ENABLED(Z_PROBE_ALLEN_KEY)
```

Comment:
This probably belonged in this file anyway... even if it was _commonly_ used by delta printers, there's no reason it couldn't be used by other types with the right config.

Reply:
I was working with a vendor last year that used this probe type on a cartesian machine, so they will probably be happy that they no longer have to port it over/out of delta configs each update.